tests/models/musicgen/test_modeling_musicgen.py

```diff
@@ -1095,10 +1097,14 @@ def test_sdpa_can_dispatch_on_flash(self):
             self.skipTest(reason="Model architecture does not support attentions")
 
         torch.compiler.reset()
-        compute_capability = torch.cuda.get_device_capability()
-        major, _ = compute_capability
 
-        if not torch.version.cuda or major < 8:
+        valid_compute_capability = False
+        if IS_CUDA_SYSTEM or IS_ROCM_SYSTEM:
+            compute_capability = torch.cuda.get_device_capability()
+            major, _ = compute_capability
+            valid_compute_capability = major < 8
```

Comment:
IIRC the logic, the valid one is major >= 8.

Reply:
not sure if the number `8` is the correct number that should be used for ROCM however.